tests/platform_tests/api/test_fan.py

```diff
@@ -91,6 +140,7 @@ def test_get_serial(self, duthost, localhost, platform_api_conn):
 
             if self.expect(serial is not None, "Unable to retrieve fan {} serial number".format(i)):
                 self.expect(isinstance(serial, STRING_TYPE), "Fan {} serial number appears incorrect".format(i))
+                self.compare_value_with_device_facts('serial', serial)
```

Comment:
`serial` will vary by device, so we are not expecting this value to be stored in the platform.json file. We should not perform a comparison here.

Reply:
removed, 
so for any device example fan, fan_drawer etc  we will not compare these values from inventory file ? model  and serial number ?

Reply:
I misread that you were gathering this value from the device facts, not the platform facts. However, we have not yet defined a schema for storing these values in the inventory file. This can be added back at a later time if we decide to store the values there.